### PR TITLE
[Quantum] Fix wrong test yml parameter (no sdk change)

### PR DESCRIPTION
**Pull request by xinyi-joffre**
This test.yml is using an old parameter name, so the build is currently broken. This PR updates to the latest name, per:
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-sdk-client.yml

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

**Comment by xinyi-joffre** [comment]
@pallavit, it looks like checkenforcer is stuck in pending--is it waiting for your approval? Thanks for your help!

**Comment by pallavit** [comment]
check-enforcer was not stuck on approval - it was not able to match your PR against any pipelines where it could runt he tests. I triggered the release of the test pipeline which did the necessary validation.

**Comment by pallavit** [comment]
@hallipr the prepare-release output looks odd to me. I expected it to trigger test runs can you please help validate this was indeed correct?

**Pull request merged**
